### PR TITLE
Quang/fix bug auto register

### PR DESCRIPTION
**Pull request by nngu6036**

### CODE REVIEW

File: collective/pfg/signup/tests/test_adapter.py [hunk 1]
```diff
@@ -23,10 +23,135 @@ def test_creation(self):
...
+        fields = self.form._getFieldObjects()
+        request = self.LoadRequestForm(fullname='test name', username='test1', email='test@mail.com',department='IT' )
+        signup.onSuccess(fields, request)
+        groups = api.group.get_groups(username='test1')
+        self.assertTrue('IT' in [group.id for group in groups],'User not created with IT group')
+
+    def test_invalid_domain_name_validation(self):
+        self.form.signup.user_group_template = Expression('string:staff')
+        self.form.signup.email_domain_verification = True
+        signup = self.form.signup
```
Comment (djay) [inline collective/pfg/signup/tests/test_adapter.py]: even though you preset the data I think to add an assert here to show the preset data so you can read the test and understand what the preset data is. ie assert the group domain = mygroup@mail.com

File: collective/pfg/signup/tests/test_adapter.py [hunk 2]
```diff
@@ -23,10 +23,135 @@ def test_creation(self):
...
+        fields = self.form._getFieldObjects()
+        request = self.LoadRequestForm(fullname='test name', username='test2', email='test@plone.org')
+        signup.onSuccess(fields, request)
+        user = api.user.get(username='test2')
+        self.assertIsNotNone(user, 'Fail to validate valid user email')
+
+    def test_approve_user(self):
```
Comment (djay) [inline collective/pfg/signup/tests/test_adapter.py]: we also need a browser test for this form as I can't see it in the UAT site so something is not working

Comment (djay) [reply collective/pfg/signup/tests/test_adapter.py]: browser test or robot test but doesn't need to reproduce all the same test cases. Just show you can see the settings for and the approval form and maybe action them

Comment (nngu6036) [reply collective/pfg/signup/tests/test_adapter.py]: Browser test added

**Pull request merged**